IndexedDB/idbfactory-open-opaque-origin.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>IDBFactory.open() and opaque origins</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script>
+
+function load_iframe(src, sandbox) {
+  return new Promise(resolve => {
+    const iframe = document.createElement('iframe');
+    iframe.onload = () => { resolve(iframe); };
+    if (sandbox)
+      iframe.sandbox = sandbox;
+    iframe.srcdoc = src;
+    iframe.style.display = 'none';
+    document.documentElement.appendChild(iframe);
+  });
+}
+
+function wait_for_message() {
+  return new Promise(resolve => {
+    self.addEventListener('message', function listener(e) {
+      resolve(e.data);
+      self.removeEventListener('message', listener);
+    });
+  });
+}
+
+const script =
+  '<script>' +
+  '  window.onmessage = () => {' +
+  '    try {' +
+  '      const r = indexedDB.open("opaque-origin-test");' +
+  '      r.onupgradeneeded = () => { r.transaction.abort(); };' +
```

Comment:
Can this ever happen if running with a fresh profile? If not, is there some test setup that could be added to always start from the state state?